kubernetes/istio/istio/src/main/java/org/arquillian/cube/istio/impl/IstioAssistant.java

```diff
@@ -0,0 +1,137 @@
+package org.arquillian.cube.istio.impl;
+
+import io.github.lukehutch.fastclasspathscanner.FastClasspathScanner;
+import io.github.lukehutch.fastclasspathscanner.matchprocessor.FileMatchProcessor;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URL;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Function;
+import me.snowdrop.istio.api.model.IstioResource;
+import me.snowdrop.istio.client.IstioClient;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.Response;
+import org.arquillian.cube.kubernetes.impl.utils.ResourceFilter;
+import org.awaitility.Awaitility;
+
+public class IstioAssistant {
+
+    private final IstioClient istioClient;
+    private final OkHttpClient httpClient;
+
+    public IstioAssistant(IstioClient istioClient) {
+        this.istioClient = istioClient;
+        this.httpClient = new OkHttpClient();
+    }
+
+    public List<IstioResource> deployIstioResources(final InputStream inputStream) {
+        return istioClient.registerCustomResources(inputStream);
+    }
+
+    public void undeployIstioResources(final List<IstioResource> istioResources) {
```

Comment:
`undeploy` -> `unDeploy`?

Reply:
I remember that I checked some time ago and it was a single word. In fact if you check VertX it uses `undeploy` http://vertx.io/docs/apidocs/io/vertx/core/Vertx.html#undeploy-java.lang.String-